app/services/pii/cipher.rb

```diff
@@ -0,0 +1,65 @@
+module Pii
+  class Cipher
+    include Encodable
+
+    def initialize
+      @cipher = OpenSSL::Cipher.new 'aes-256-gcm'
```

Comment:
I'm having trouble following if `Pii:Cipher` instances are reused. It's probably easier if you just create a new cipher instance in `encrypt` and `decrypt`, since they mutate the instance and this will make reasoning about this in the face of concurrency much simpler.